modules/ROOT/partials/plugins/comments-open-sidebar.adoc

```diff
@@ -38,12 +38,7 @@ tinymce.init({
   tinycomments_mode: 'embedded',
   tinycomments_author: currentAuthor,
   tinycomments_can_resolve: canResolveCommentsCallback,
-  /* The following setup callback opens the comments sidebar when the editor loads */
-  setup: (editor) => {
-    editor.on('SkinLoaded', () => {
-      editor.execCommand("ToggleSidebar", false, "showcomments");
-    })
-  }
+  sidebar_show: 'showcomments',
```

Comment:
Shouldn't we also update the above code snippet and sentence at the beginning of the 'Show the comments sidebar when TinyMCE loads' section?